Orange/widgets/unsupervised/owmanifoldlearning.py

```diff
@@ -93,11 +93,29 @@ class TSNEParametersEditor(ManifoldParametersEditor):
     # rename l2 to Euclidean
     metric_values = [("l2", "Euclidean")] + metric_values
 
+    perplexity = Setting(30)
+    early_exaggeration = Setting(4)
+    learning_rate = Setting(1000)
+    n_iter = Setting(1000)
+
+    init_index = Setting(0)
+    init_values = [("random", "Random"), ("pca", "PCA")]
```

Comment:
Can we leave the default init to be pca.
I would also switch their positions to have pca as the first (default) option and then random.
This would then also make it consistent with mds (pca = first & default)

Reply:
I know this goes against the sklearn defaults, but we already had pca as the only option before and I think it makes more sense for users (random can show a completely different figure each time). The consistency with mds is an added bonus.

Reply:
As far as I can tell, both the MDS widget and manifold MDS control pane have the option to initialize randomly. I have, however, made PCA the default value, since that is what MDS does in both cases. Is there a third MDS widget that I missed?

Reply:
This is fine now. That is what I meant - just change the order of pca and random, so that pca is default (but we still have both options, like in mds)